string-constants-doc/string-constants/translation-guide.scrbl

```diff
@@ -0,0 +1,161 @@
+#lang scribble/manual
+
+@title{How To Translate DrRacket's String Constants in Your Language}
+@author[(author+email "Alexander Shopov" "ash@kambanaria.org"  #:obfuscate? #t)]
+
+This is a short hands on guide how to translate the strings used
+within DrRacket's GUI. It is targeted at people willing to translate
+the interface who may not know all ins and outs of Racket or Scheme.
+
+@; ----------------------------------------------------------------------
+
+@section{Introduction}
+
+DrRacket is a GUI: editor and runner for the many languages you can
+use via Racket — the language developer’s language. Currently
+existing translations are English Danish, French, Finnish, German,
+Portuguese, Russian, Ukrainian, Bulgarian, Chinese (traditional and
+simplified) and Japanese. Dutch, Korean and Spanish have not been
+updated in a long while and they still need relicensing which makes
+them hidden in the interface.
+
+There are many languages missing plus some of the existing
+translations need some work for updates.
+
+If you are interested how much time and effort you will need to
+invest– currently it is roughly a 1400 string translation, of less
+than 6000 words — it is an easy-peasy deal.
```

Comment:
You can compute these numbers as it is Scribble. :)

Reply:
Yep, I know - I have to read the file and then basically count the keys and the number of words in the Englishvalues. TBH I just did a quick sed/wc combo to get the rough numbers. The numbers are bound to change. I'll take that as a homework assignment.

Reply:
We could change the string constants language to export this information or just saying "roughly three afternoons worth of work for an experienced translator" and leave it at that, maybe. I'm happy either way and happy to assist if you want to do the more precise thing.